From 464824279827b5188abd969ee754609c22f4e393 Mon Sep 17 00:00:00 2001
From: David Faure <faure@kde.org>
Date: Tue, 23 Jul 2019 11:23:01 +0200
Subject: [PATCH] Fix the way poppler headers are included.

The pkg config file already has the poppler subdir in -I
---
 filters/karbon/pdf/PdfImport.cpp    | 5 +++--
 filters/karbon/pdf/SvgOutputDev.cpp | 8 ++++----
 filters/karbon/pdf/SvgOutputDev.h   | 5 +++--
 3 files changed, 10 insertions(+), 8 deletions(-)

diff --git a/filters/karbon/pdf/PdfImport.cpp b/filters/karbon/pdf/PdfImport.cpp
index e910dae0756..f88719d4554 100644
--- a/filters/karbon/pdf/PdfImport.cpp
+++ b/filters/karbon/pdf/PdfImport.cpp
@@ -35,8 +35,9 @@
 #pragma GCC diagnostic ignored "-Wunused-parameter"
 #endif
 
-#include <poppler/PDFDoc.h>
-#include <poppler/GlobalParams.h>
+// poppler includes
+#include <PDFDoc.h>
+#include <GlobalParams.h>
 
 K_PLUGIN_FACTORY_WITH_JSON(PdfImportFactory, "calligra_filter_pdf2svg.json",
                            registerPlugin<PdfImport>();)
diff --git a/filters/karbon/pdf/SvgOutputDev.cpp b/filters/karbon/pdf/SvgOutputDev.cpp
index ae2538a2e03..a4e23841e37 100644
--- a/filters/karbon/pdf/SvgOutputDev.cpp
+++ b/filters/karbon/pdf/SvgOutputDev.cpp
@@ -21,10 +21,10 @@
 
 #include "PdfImportDebug.h"
 
-#include <poppler/Object.h>
-#include <poppler/GfxState.h>
-#include <poppler/Stream.h>
-#include <poppler/GfxFont.h>
+// Poppler includes
+#include <GfxState.h>
+#include <Stream.h>
+#include <GfxFont.h>
 
 #include <QFile>
 #include <QTextStream>
diff --git a/filters/karbon/pdf/SvgOutputDev.h b/filters/karbon/pdf/SvgOutputDev.h
index f0d180fd7dd..2716d5e95a4 100644
--- a/filters/karbon/pdf/SvgOutputDev.h
+++ b/filters/karbon/pdf/SvgOutputDev.h
@@ -25,8 +25,9 @@
 #pragma GCC diagnostic ignored "-Wunused-parameter"
 #endif
 
-#include <poppler/Object.h>
-#include <poppler/OutputDev.h>
+// poppler includes
+#include <Object.h>
+#include <OutputDev.h>
 
 #include <QString>
